scrapers/FemdomEmpire.yml

```diff
@@ -60,6 +68,14 @@ xPathScrapers:
           - replace:
             - regex: "(^[^|]+)\\|/tour/([^\\.]+\\.jpg).*"
               with: $1$2
-
+  sceneSearch:
+    common:
+      $itemThumb: //div[@class='item-thumb']
+    scene:
+      Title: $itemThumb/a/@title
+      URL: $itemThumb/a/@href
+      Image: $itemThumb/a/img/@src
+driver:
+  useCDP: true
```

Comment:
Adding CDP is not needed and it makes the whole scraper a lot slower
You just need to get the correct selectors from page source eg
```yaml
      Image:
        selector: $itemThumb/a/img/@*[name()='src0' or name()='src0_1x']
        postProcess:
          - replace:
              - regex: ^/
                with: https://femdomempire.com/
```
seems to work for me